python/dask_cudf/dask_cudf/io/opt_parquet.py

```diff
@@ -124,18 +124,12 @@ def _read_partition(part, index, columns, strings_to_cats):
     if isinstance(index, list):
         columns += index
 
```

Comment:
Note that the primary `read_parquet` code path for `dask_cudf` is in a separate file (you cant think of the code in `opt_parquet.py` as a temporary/optimized code path to avoid some slow upstream behavior when the user passes in `row_groups_per_part`).  The official path is still passing `row_group` in a few places [like here](https://github.com/rapidsai/cudf/blob/1a52851a0b447b26318602d6f4de8e8b4febdfe0/python/dask_cudf/dask_cudf/io/parquet.py#L55) (which is likely causing the CI failures).

Reply:
found it, thanks!